acceptance_tests/basic/src/lib.rs

```diff
@@ -147,17 +147,17 @@ mod test_cases {
     #[derive(Debug)]
     #[allow(dead_code)]
     enum SimpleEnum {
-        Var1(i32),
```

Comment:
Why did you touch this enum?

Reply:
I had to due to `quote!` and `Debug` combination producing different outputs on different rust versions for ADTs.
Right now we have that space between `Var` and `(_, 5)` that we test for but it is not there for `1.41` and above.

Reply:
I can't think of a way to test for ADTs with `panic` keyword sadly, until I can guarantee that the message produced will have consistent text across versions.